src/js/common/schemaform/widgets/SSNWidget.jsx

```diff
@@ -11,7 +11,11 @@ export default class SSNWidget extends React.Component {
     this.state = { val: props.value };
   }
   handleChange = (val) => {
-    const strippedSSN = val.replace(/[\- ]/g, '');
+    let strippedSSN;
+    if (val) {
+      strippedSSN = val.replace(/[\- ]/g, '');
+    }
```

Comment:
Perhaps having this in `handleChange` isn't the best place, but is it needed elsewhere? I mean, when we make a new component with `val=''` is the only time that it can be `''` without getting translated to `undefined`.

However, it also means the unit test that renders an empty string still works, but is perhaps misleading.